pkg/lib/operatorclient/client.go

```diff
@@ -127,6 +129,18 @@ type DeploymentClient interface {
 	ListDeploymentsWithLabels(namespace string, labels labels.Set) (*appsv1.DeploymentList, error)
 }
 
+type WebhookClient interface {
```

Comment:
don't block on this, but we've been moving away from operatorclient and towards the standard kube client, so this isn't really needed here.

Reply:
Updated the code to use the kube client.